src/main/java/by/onliner/constants/OnlinerConstants.java

```diff
@@ -6,4 +6,16 @@
 
     public static final String LOG_PATH = "target/logs/appTest.log";
     public static final String SAVE_VIDEO_PATH = "./Videos/";
+
+    public static final String ABOUT_LINK = "/about";
+
+    public static final String IPHONE = "iphone 13";
+
+    public static final String INCORRECT_LOG_IN_WARNING_TEXT = "укажите ник или e-mail";
+    public static final String INCORRECT_PASSWORD_WARNING_TEXT = "укажите пароль";
```

Comment:
it's better to store this warning text in the class with the tests where it's used as this is not just a constant but also test data.
I don't think it will be used anywhere but login tests

Reply:
Moved to the test class, in the future I plan to create test data